README.md

```diff
@@ -306,6 +306,82 @@ function main() {
 }
 ```
 
+## Example Code for 24 Segment Bargrah
+
+![](docs/24segments.gif)
+
+
+```
+import { connect8x8BicolorMatrix } from '@thingssdk/ht16k33/espruino';
```

Comment:
connect8x8BicolorMatrix is for Bi Color Displays

A more appropriate API would be BicolorBargraph